### PR TITLE
Fix the stop sound effect pattern

### PR DESCRIPTION
**Pull request by TheLimeGlass**
### Description
Fix the stop sound effect pattern.

---
**Target Minecraft Versions:** 1.17+
**Requirements:** none
**Related Issues:** https://github.com/SkriptLang/Skript/issues/5756

### CODE REVIEW

**Review by Moderocky — APPROVED** [review]

**Comment by TheLimeGlass** [comment]
The 2.7 target of this pull was merged. Waiting on another approval. Any requested changes will require another 2.7 target pull request to clone changes.

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]

**Review by sovdeeth — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffStopSound.java
Comment (UnderscoreTud) [reply src/main/java/ch/njol/skript/effects/EffStopSound.java:63]: The expression isn't optional therefore making it doesn't make sense

Comment (TheLimeGlass) [reply src/main/java/ch/njol/skript/effects/EffStopSound.java:63]: A string could be anything from the event. I don't even think we have a sound string as a default value. This would not work.

**Review by APickledWalrus — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffStopSound.java
Comment (TheLimeGlass) [reply src/main/java/ch/njol/skript/effects/EffStopSound.java:63]: This was already merged into 2.7, any changes are separate. I made both the 2.7 target and this at the same time, the 2.7 target got merged first I guess.

**Comment by TheLimeGlass** [comment]
Closing because this got merged in 2.7 and was at the time when the release cycle changed, so it got added to feature branch.

**Pull request closed without merging**
